netci.groovy

```diff
@@ -115,18 +115,18 @@ def CreateBuildTasks = { machine, configTag, buildExtra, testExtra, runCodeAnaly
 
 def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSetup ->
     [true, false].each { isPR ->
-        ['debug', 'release'].each { buildType ->
+        ['debug', 'release', 'test-build'].each { buildType ->
```

Comment:
Hmm. naming difference can be handled below though. I mean that part can be 'test' and if 'test' then use 'test-build' flag etc. IMO also better. Let me update the PR.